gobin/exe.go

```diff
@@ -37,36 +38,77 @@ func toPackages(ctx context.Context, out *[]*claircore.Package, p string, r io.R
 
 	// TODO(hank) This package could use canonical versions, but the
 	// [claircore.Version] type is lossy for pre-release versions (I'm sorry).
+	var runtimeVer claircore.Version
```

Comment:
Judging by the comment, this needs more thought if we're going to correctly index pre-release versions